test/fixtures/bpmn/model-complete-semantic.bpmn

```diff
@@ -158,6 +158,18 @@
       <semantic:multiInstanceLoopCharacteristics />
     </semantic:scriptTask>
 
+    <!-- Business Rule Task   -->
```

Comment:
Missing the **Diagram** part of the corresponding semantic objects

Reply:
The update of the corresponding e2e test is missing : **test/e2e/mxGraph.model.test.ts** -> i`t('bpmn elements should be available in the mxGraph model'`